includes/wf_crm_webform_postprocess.inc

```diff
@@ -2131,7 +2131,7 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
           continue;
         }
         // Ignore values from fields hidden by existing contact component
-        if ($this->isFieldHiddenByExistingContactSettings($ent, $c, $table, $n, $name)) {
```

Comment:
Update the comment to something like "Check to see if configured to ignore hidden values"